rust/psibase/src/tester.rs

```diff
@@ -294,16 +273,27 @@ impl<T: fracpack::UnpackOwned> ChainResult<T> {
             let ret = transact
                 .inner_traces
                 .iter()
+                // TODO: improve this filter.. we need to return whatever is the name of the action somehow if possible...
```

Comment:
I think this comment is outdated. The output already *does* get the name of the action. Sample output:
```
>>> ChainResult::get - Inner action trace: setcpulimit (raw_retval=)
```

Reply:
Problem is that we dont know which action name we should filter by...